frontend/src/common/helpers/helpers.tsx

```diff
@@ -0,0 +1,24 @@
+// Get the openshift environment name from the url
+// Example:
+// https://onroutebc-test-frontend.apps.silver.devops.gov.bc.ca
+// hostname = onroutebc-test-frontend.apps.silver.devops.gov.bc.ca
+// env = ["-test-", "test"]
+// env[1] = "test"
+export const getOpenshiftEnv = () => {
+  const url = window.location.href;
+  const { hostname } = new URL(url);
+  const env = hostname.match("-(.*)-");
```

Comment:
Word of caution; this might work for internal domains but I doubt if we'll use `onroutebc-prod-frontend`  for production.  In production it might just be `onroutebc-frontend`.

Reply:
Agreed. I have updated the logic to account for the url in prod. In the future we will move away from using `window.location.href` to get the openshift envirornment, but instead use a .env variable or make a call to the backend.